src/pages/Followers.jsx

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios'
+import { useEffect, useState } from 'react'
+import { FeedPageWrapper, PageHeading } from "components/Reusable"
+import styles from 'components/Reusable/post.module.css'
+import { useSelector } from 'react-redux'
+import { useNavigate, useParams } from 'react-router-dom'
+
+const FollowersSection = () => {
+    const [followers, setFollowers] = useState([])
+    const navigate = useNavigate()
+    const params = useParams()
+
+    useEffect(() => {
+        (async () => {
+            const response = await axios.get('/api/users')
+            const currentUser = response.data.users.find(user => user.username === params.username)
+            setFollowers(currentUser.followers)
+        })()
+    }, [params.username])
+
+    return (
+        <div className={`${styles.feedDiv} flx flx-column pd-md`}>
+            <PageHeading heading='followers' />
+            {
+                followers?.map(follower =>
+                    <article onClick={() => navigate(`/${follower.username}`)} className={`${styles.followerDiv} flx flx-min-center pd-btm-xs mg-s`}>
```

Comment:
Suggestion - Use some formatter for code formating , because the code goes ouside the view, one need to scroll

Reply:
I am using the formatter.